### PR TITLE
TTH-6 navbar

### PR DESCRIPTION
**Pull request by kirtanmedi**
* **Please check if the PR fulfills these requirements**
- [ ] The pull request title contains the ticket number in the title. For example [CB-1]

* **What kind of change does this PR introduce?** (Bug fix, new feature...)

* **What is the current behavior?What does the code do before you made the new changes?** 

* **What is the new behavior?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)

* **Other information**:

### CODE REVIEW

File: frontend/src/components/Navigationbar.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  Nav,
+  NavItem,
+  NavLink,
+  NavbarText
+} from 'reactstrap';
+
+const Navigationbar = (props) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggle = () => setIsOpen(!isOpen);
```
Comment (Codubee) [inline frontend/src/components/Navigationbar.js]: Lets try to keep these to either functional or class components. [Here](https://imgur.com/CO0B4U6) is an example of how to do that.

File: frontend/src/components/Navigationbar.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+  return (
+    <div>
+      <Navbar color="dark" dark expand="md">
+        <NavbarBrand href="/">
+          TranslateNews&nbsp;
```
Comment (Codubee) [inline frontend/src/components/Navigationbar.js]: What does &nbsp do?

File: frontend/public/index.html [hunk 3]
```diff
@@ -9,6 +9,9 @@
       name="description"
       content="Web site created using create-react-app"
     />
+
+    <script src="https://kit.fontawesome.com/71bead3b94.js" crossorigin="anonymous"></script>
```
Comment (Codubee) [inline frontend/public/index.html:13]: What does this do?

**Review by Codubee — CHANGES_REQUESTED** [review]
Great work Kirtan. Left some comments to discuss.

File: frontend/public/index.html (hunk 3 above)
Comment (kirtanmedi) [reply frontend/public/index.html:13]: This allows me to pull various icons and texts from a font website. https://fontawesome.com/

Comment (Codubee) [reply frontend/public/index.html:13]: Ah ok cool!

File: frontend/src/components/Navigationbar.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        </NavbarBrand>
+        <NavbarToggler onClick={toggle} />
+        <Collapse isOpen={isOpen} navbar>
+          <Nav className="mr-auto" navbar>
+            <NavItem>
+              <NavLink href="/">News</NavLink>
+            </NavItem>
+          </Nav>
+          <NavbarText><i class="fas fa-language"></i></NavbarText>
+          <NavbarText>&nbsp;<i class="fas fa-newspaper"></i></NavbarText>
```
Comment (Codubee) [inline frontend/src/components/Navigationbar.js]: Just wondering what these two are for.

File: frontend/src/components/Navigationbar.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+function Navigationbar(){
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggle = () => setIsOpen(!isOpen);
+
+  return (
+    <div>
+      <Navbar color="dark" dark expand="md">
+        <NavbarBrand href="/">
+          TranslateNews&nbsp;
```
Comment (Codubee) [inline frontend/src/components/Navigationbar.js]: Remove the nbsp please.

**Review by Codubee — CHANGES_REQUESTED** [review]
Left some more comments.

**Review by Codubee — APPROVED** [review]

**Pull request merged**